cmake/extensions.cmake

```diff
@@ -2788,7 +2788,7 @@ function(dt_reg_index_private var path name)
   else()
     list(FIND reg_names "${name}" index)
   endif()
-  set("${var}" "${index}" PARENT_SCOPE)
+  set(${var} "${index}" PARENT_SCOPE)
```

Comment:
ahh, here's the reason.
I believe this should be in previous commit.

Reply:
> I believe this should be in previous commit.

@tejlmand You're right, of course. Fixed.